.github/workflows/ci.yaml

```diff
@@ -8,7 +8,12 @@ on:
   push:
     branches:
       - "renovate/**"
-  
+
+# Prevent duplicate runs if Renovate falls back to creating a PR
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref_name }}
```

Comment:
Is it possible to not have a head_ref?

Reply:
On a PR it's not the branch of origin so it was not grouped together
![image](https://github.com/user-attachments/assets/5988e8a4-3fa8-4136-96e7-047b6c57b9d3)
![image](https://github.com/user-attachments/assets/6fbfdd79-a8e4-4954-a736-5fe58ddb9518)

Reply:
Try some, no success: 
![image](https://github.com/user-attachments/assets/d76547f0-085d-401e-935e-5d157efd905c)